mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+          properties:
+             - !ruby/object:Api::Type::String
+               name: 'column'
+               description: |
+                 The unnested column which this rule is evaluated against.
+             - !ruby/object:Api::Type::Boolean
+               name: 'ignoreNull'
+               description: |
+                 Rows with null values will automatically fail a rule, unless ignoreNull is true. In that case, such null rows are trivially considered passing. Only applicable to ColumnMap rules.
+             - !ruby/object:Api::Type::String
+               name: 'dimension'
+               required: true
+               description: |
+                 The dimension a rule belongs to. Results are also aggregated at the dimension level. Supported dimensions are ["COMPLETENESS", "ACCURACY", "CONSISTENCY", "VALIDITY", "UNIQUENESS", "INTEGRITY"]
+             - !ruby/object:Api::Type::Integer
+               name: 'threshold'
+               description: |
+                The minimum ratio of passing_rows / total_rows required to pass this rule, with a range of [0.0, 1.0]. 0 indicates default value (i.e. 1.0).
+               default_value: 0
+             - !ruby/object:Api::Type::NestedObject
+               name: 'rangeExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value lies between a specified range.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'minValue'
+                   description: |
+                     The minimum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::String
+                   name: maxValue
+                   description: |
+                     The maximum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMinEnabled'
+                   default_value: false
+                   description: |
+                     Whether each value needs to be strictly greater than ('>') the minimum, or if equality is allowed. 
+                     Only relevant if a minValue has been defined. Default = false.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMaxEnabled'
+                   default_value: false
+                   description: |
+                     Whether each value needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                     Only relevant if a maxValue has been defined. Default = false.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'nonNullExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value is null.
```

Comment:
are we missing a nested `values` list of strings here? it should be `required` as its the only nested attribute

Reply:
This is another example of an empty message in the API; We may add future fields / properties to this message later, but it's currently empty;